test/test_helper.rb

```diff
@@ -10,6 +10,11 @@
 #
 ################################################################################
```

Comment:
Tells minitest to broadcast test results to any IDE's that interested in them.  Makes this place nicely with test runners like that integrated in RubyMine